xwiki-pro-macros-ui/src/main/resources/XWiki/Macros/Translations.xml

```diff
@@ -29,6 +29,7 @@
   <creator>xwiki:XWiki.Admin</creator>
   <parent>xwiki:XWiki.Macros.WebHome</parent>
   <author>xwiki:XWiki.Admin</author>
+  <originalMetadataAuthor>XWiki.superadmin</originalMetadataAuthor>
```

Comment:
I think you need to omit this line as it's not supported / known by older versions of XWiki.

Reply:
I removed this line indeed.